### PR TITLE
N51: add rustunit bevy meetup 4

### PR DESCRIPTION
**Pull request by Vrixyz**
_Part of #1505_

### CODE REVIEW

File: content/news/051/index.md [hunk 1]
```diff
@@ -128,15 +129,29 @@ _Bevy meetup youtube extract, showing its 3 participants._
 - Julian - Animating 3D Characters with Bevy
 - Niklas - Learnings from a Bevy game template
 
-Check out [the recording][rustunit_bevy_meetup_youtube]
+Check out [the recording][rustunit_bevy_meetup_3_youtube]
+and join [the meetup group][rustunit_bevy_meetup_event] to get notified of future events.
+
+[rustunit_bevy_meetup_3_youtube]: https://www.youtube.com/watch?v=Fyr1ud1OIpU
+
+### Bevy rustunit meetup #4
```
Comment (Vrixyz) [inline content/news/051/index.md:137]: Gotta verify how the <pre>#4</pre> renders

**Comment by janhohenheim** [comment]
Gonna review this tomorrow and check out the render 👍

File: content/news/051/index.md (hunk 1 above)
Comment (Vrixyz) [reply content/news/051/index.md:137]: seems fine, commenting the render in thread

**Comment by Vrixyz** [comment]
<details><summary>Render screenshot</summary>
<p>

Screenshot 2024-05-26 at 19 34 51

</p>
</details>

**Pull request merged**